### PR TITLE
Migration to RoomCode

### PR DESCRIPTION
**Pull request by kushal7201**
Issue: #42 

I have updated the controllers for migration from RoomId to RoomCode.

### CODE REVIEW

File: src/controllers/room.controllers.js [hunk 1]
```diff
@@ -80,27 +80,27 @@ async function createRoom(req, res) {
...
             });
             return;
         }
-
+ 
         const userOwnsRoom = await prisma.user.findUnique({
             where: {
                 id: req.user.id,
-                userRoomId: roomId,
+                userRoomCode: roomCode,
```
Comment (VatsalBhuva11) [inline src/controllers/room.controllers.js]: There's no such thing as "userRoomCode" in the User model.

File: src/controllers/room.controllers.js [hunk 2]
```diff
@@ -111,90 +111,90 @@ async function updateRoom(req, res) {
...
             });
             return;
         }
-
+ 
         const userOwnsRoom = await prisma.user.findUnique({
             where: {
                 id: req.user.id,
-                userRoomId: room.roomId,
+                userRoomCode: roomCode,
```
Comment (VatsalBhuva11) [inline src/controllers/room.controllers.js]: Same as above.

File: src/controllers/room.controllers.js [hunk 3]
```diff
@@ -950,104 +950,104 @@ export const announce = async (req, res) => {
...
+     }
+   
+     return response_200(res, 'Users fetched successfully', {
+         users: room.users.map((user) => {
+             return {
+                 id: user.id,
+                 name: user.name,
+                 email: user.email,
+                 currPoints: user.currPoints,
+                 roomCode: user.userRoomId,
```
Comment (VatsalBhuva11) [inline src/controllers/room.controllers.js:1042]: I think this is incorrect. Please correct it.

**Comment by kushal7201** [comment]
@VatsalBhuva11  I have updated the changes

File: src/controllers/room.controllers.js [hunk 4]
```diff
@@ -80,27 +80,27 @@ async function createRoom(req, res) {
...
             });
             return;
         }
-
+ 
         const userOwnsRoom = await prisma.user.findUnique({
             where: {
                 id: req.user.id,
-                userRoomId: roomId,
+                roomCode: roomCode,
```
Comment (VatsalBhuva11) [inline src/controllers/room.controllers.js:103]: There is no `roomCode` as well in the `User` model. Verify with the schema and make these corrections everywhere for the User model

**Pull request closed without merging**